.circleci/config.yml

```diff
@@ -168,11 +165,11 @@ jobs:
       - run:
           name: Build example APKs
           command: |
-            melos run build:examples:android:<< parameters.plugin >>
+            melos run build:example:android << parameters.plugin >>
           no_output_timeout: 20m
-      - run:
-          name: Run lint checks
-          command: melos run lint:android:<< parameters.plugin >>
+      # - run:
```

Comment:
At some point I want to rethink it from scratch - the iOS config has not been updated in a while and has invalid options which throw errors because of that. I have an opinion now on how things should look and will work on reintroducing this before launch.